model/qti/feedback/ModalFeedback.php

```diff
@@ -47,18 +41,14 @@ class ModalFeedback extends Feedback
 
     protected function getUsedAttributes()
     {
-
         return array_merge(
             parent::getUsedAttributes(),
-            [
-            'oat\\taoQtiItem\\model\\qti\\attribute\\TitleOptional'
-                ]
+            [TitleOptional::class]
         );
     }
     
     public function toForm()
     {
-        $formContainer = new ModalFeedback($this);
-        return $formContainer->getForm();
+        return (new self($this))->getForm();
     }
 }
```

Comment:
Are we using this method?
And I think that this one won't work, its parents `__construct` expects an array, we will get the error here: https://github.com/oat-sa/extension-tao-itemqti/blob/ee1d800d983e1cb2bac8fc0e7dbb5b2465c36877/model/qti/Element.php#L139

Reply:
I am not sure that it is actually used somewhere, but probably yes. I am fixed only code styles and issues, that IDE highlighted by red across all files. But yes, it makes sense to remove it as unused.

Reply:
I removed the method as unused.